README.md

```diff
@@ -12,6 +12,85 @@ with Zipkin-compatible data model.
+
+```javascript
+    import { TChannelBridge } from 'jaeger-client';
+
+    let bridge = new TChannelBridge(tracer);
+    let server = new TChannel({ serviceName: 'server' });
+    server.listen(4040, '127.0.0.1');
+    let serverThriftChannel = TChannelAsThrift({
+        channel: server,
+        entryPoint: path.join(__dirname, 'thrift', 'echo.thrift') // file path to a thrift file
+    });
+
+    serverThriftChannel.register(server, 'Echo::echo', context, bridge.tracedHandler(
+        (context, req, head, body, callback) => {
+            // context will contain an 'openTracingSpan' field that stores the tracing span for the inbound request.
+        }
+    ));
+```
+
+
+In the case of making an outgoing request you can wrap an encoded channel in a call to `tracedChannel`.
+
+```javascript
+    import { TChannelBridge } from 'jaeger-client';
+
+    let bridge = new TChannelBridge(tracer);
+    // Create the toplevel client channel.
+    let client = new TChannel();
+
+    // Create the client subchannel that makes requests.
+    let clientSubChannel = client.makeSubChannel({
+        serviceName: 'server',
+        peers: ['127.0.0.1:4040']
+    });
+
+    let encodedThriftChannel = TChannelAsThrift({
+        channel: clientSubChannel,
+        entryPoint: path.join(__dirname, 'thrift', 'echo.thrift') // file path to a thrift file
+    });
+
+    // The 'context' object must be passed through from the request method with the field name 'openTracingContext' to ensure an uninterrupted trace.
+    // If the context is empty, a new trace will be started for the outbound call.
+    // The 'openTracingContext' object must also have an 'openTracingSpan' field that represents the current span.
+    let tracedChannel = bridge.tracedChannel(encodedThriftChannel);
+
+    // The encodedThriftChannel's (also true for json encoded channels) request object can call 'send' directly.
+    let req = tracedChannel.request({
+        serviceName: 'server',
+        openTracingContext: { openTracingSpan: span }, // where span is the current context's span
+        headers: { cn: 'echo' }
+    });
+
+    // headers should contain your outgoing tchannel headers if any.
+    // In this instance 'send' is being called on the request object, and not the channel.
+    req.send('Echo::echo', headers, { value: 'some-string' });
+```
+
+It should be noted that if you intend to call tchannel's `send` method with the alternate style
+of calling 'send', then you need to create the request object using the top level channel's request method.
+
+```javascript
+    let tracedChannel = bridge.tracedChannel(encodedThriftChannel);
+
+    // tracedChannel.channel refers to encodedThriftChannel's inner channel which
+    // is clientSubChannel in this instance.
+    let req = tracedChannel.channel.request({
```

Comment:
Users should be able to wrap encoded channel in traced channel once, and not have to change the rest of their code (aside from passing context)